src/category_theory/groupoid/subgroupoid.lean

```diff
@@ -91,11 +130,17 @@ begin
   simp only [inv_eq_inv, is_iso.hom_inv_id],
 end
 
+lemma id_mem_of_src  {c d : C} {f : c ⟶ d} (h : f ∈ S.arrows c d) : (𝟙 c) ∈ S.arrows c c :=
+id_mem_of_nonempty_isotropy S c (mem_objs_of_src S h)
+
+lemma id_mem_of_tgt  {c d : C} {f : c ⟶ d} (h : f ∈ S.arrows c d) : (𝟙 d) ∈ S.arrows d d :=
+id_mem_of_nonempty_isotropy S d (mem_objs_of_tgt S h)
+
 /-- A subgroupoid seen as a quiver on vertex set `C` -/
 def as_wide_quiver : quiver C := ⟨λ c d, subtype $ S.arrows c d⟩
 
 /-- The coercion of a subgroupoid as a groupoid -/
-instance coe : groupoid S.objs :=
+@[simps (lemmas_only)] instance coe : groupoid S.objs :=
```

Comment:
Judging at the comment you left in the original PR, I think the `comp` lemma is fine and it is the `inv` lemma that is triggering the linter. For the former, it should still be a simp lemma. For the latter, I think there should be a simp lemma `↑(category_theory.inv p) = inv ↑p` or similar.

In this case, you might want to change the attribute to `@[simps comp_coe, simps inv_coe (lemmas_only)]`, which would generate both `comp_coe` and `inv_coe` but only the former is a simp lemma. You can change `simps` to `simps?` to check that they generate the right lemmas.